lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
> it could be reused just like minuteInterval

Somehow I see options like `maximumDate`, `minimumDate`, `minuteInterval`, `mode` different as they specify something about the data (the `t.Date` type), so they can be likely reused in different contexts.

> you could actually reuse the background prop in order to make all the touchable components behave the same

Yep, but if you *do* want to allow different behaviours you end up with a `config` API like this:

```
config = {
  removeButton: {
    label: 'X',
    background: TouchableNativeFeedback.Ripple('red') // background1
  },
  pickerButton: {
    label: '^',
    background: TouchableNativeFeedback.Ripple('green') // background2
  },
  ...
}
```

Just pass `background1 === background2 === ...` to get the same behaviour. However in this case having **also** a `background` option outside `config` would be a confusing API. 

Moreover you could write a template where there's no `TouchableNativeFeedback`s (example: a label + a slider) or a template that integrates a 3d party widget for which `background` makes no sense at all.

p.s.
Sorry, I'm pedantic and maybe I'm wasting your time with all these little details but I learned to be very careful in designing new APIs.
Go for what you feel is the best.